### ISSUE TITLE
Suggestion : Ask for issues when raised to also include an optional screen capture showingcasing the issue and it's manifestation

### ISSUE THREAD

**Issue by kilasuit** [PowerShell/PowerShell#8564]
In reading many issue's in this repo I see a smallish but nevertheless a common theme occurring.

- Badly written issues, mainly in content of the issue but often also in the technical context behind the issue that can mislead the readers into what the actual issue is
- Issues that due to being written up badly, perhaps ENG is not the writers first language so translations get lost in the midst, this can lead to long time being spent on issues and PR's intended to fix what could be completely incorrectly understood issue.

To combat this, I am suggesting

- [ ] That the Issue Template for Bug gets updated to include a message that suggests for Triage purposes that the reporter should try to include a screen capture of the issue as it's happening. This would be extremely useful for issues where this could hypothetically reduce the time taken to come to a correct diagnosis of the issue, and further confirmation that the issue is in fact correctly raised and contains the right level of detail to aid in lowering the mean time between confirmed diagnosis with potential fix identified and then the time to implement said fix.
- [ ] That other Issue Templates be updated accordingly to suggest that the above would be a beneficial addition where possible.
- [ ] If implemented then new labels should also be added to signify a screen capture was/was not included in the issue submission

I think that the above will help with longer term metrics to see how well the current triaging system works (which seems to be pretty solid thus far in the majority of cases) and if there may be any potential areas to improve further.

**Comment by vexx32** [comment]
I'd agree that people including this might be a good idea.

However, given that the majority of the issues with a problem like this seem to arise from people either not reading or not fully comprehending the issue template in the first place (whether it's due to linguistic differences or just them simply not thinking they need to read any of it), it's fairly probably that any such additions might also go similarly unheard.

**Comment by SteveL-MSFT** [comment]
I agree that there is a problem, but also agree that most users who are not regulars don't really pay attention to the template anyways.  Perhaps if we had a `PSCoreIssue` module which had a `Register-PSCoreIssue` cmdlet that walked them through a repro and collected system information/screenshot, that would be ideal :)

**Comment by vexx32** [comment]
Some things really need a screen recording on occasion though.

But yeah, the rest would be great. Even better if it could automatically submit the issue for them :wink:

**Comment by iSazonov** [comment]
[indows Problem Step Recorder](https://support.microsoft.com/en-us/help/22878/windows-10-record-steps) could be very useful if we would had a player.

**Comment by kilasuit** [comment]
This got added as the visuals section on the Bug template so closing as fixed :-)
